lib/index.js

```diff
@@ -2,6 +2,7 @@
 
 const Useragent = require('useragent');
 
+// Requires semver be installed
```

Comment:
Why do we need this comment since `semver` is already part of the dependencies in the package.json?

Reply:
I added this because semver is in scooter's dependencies, but there's no explanation why anywhere in this repo.  So I almost removed it!  Maybe I can improve the wording of this comment.

Reply:
Alright, got it. Makes sense. I guess we can leave it as is, and if someone tries to delete it again then we can see if it needs improvements.